Arrowgene.Ddon.Shared/Entity/PacketStructure/S2CPawnTrainingGetTrainingStatusRes.cs

```diff
@@ -0,0 +1,40 @@
+using Arrowgene.Buffers;
+using Arrowgene.Ddon.Shared.Network;
+
+namespace Arrowgene.Ddon.Shared.Entity.PacketStructure
+{
+    public class S2CPawnTrainingGetTrainingStatusRes : ServerResponse
+    {
+        public override PacketId Id => PacketId.S2C_PAWN_TRAINING_GET_TRAINING_STATUS_RES;
+
+        public S2CPawnTrainingGetTrainingStatusRes()
+        {
+            TrainingStatus = new byte[64];
```

Comment:
What exactly is the training data? It is really just a byte array?

Reply:
Yes. A binary blob sent by the client that the server doesn't need to understand, kind of like item sort data.